seed_helen_comments.py

```diff
@@ -0,0 +1,354 @@
+"""Utility file to seed database from pillow data/"""
+
+# import datetime
+from sqlalchemy import func
+
+from model import Show, Show_Color, Brand, Color, connect_to_db, db
+from server import app
+
+from PIL import Image, ImageDraw, ImageColor, ImageFilter
+import requests
+from StringIO import StringIO
+import os
+import sys
+from webcolors import *
+
+import pdb
+
+years = [2017]
```

Comment:
Global variables like these are usually considered bad practice.  If someone were to import this `seeds` module, and created their own `years` variable, they may get it confused with this one without knowing (or if you made a typo in any function here and typed `years` instead of `year`, your code would do unexpected things without throwing an error.

For hard coded constants, which these are, we usually use all caps (YEARS = [2017]) to denote that no one should reassign a value to that variable anywhere else.

Additionally, I would pass `years` and `seasons` directly to `feed_urls()`, like you're doing with `brands`, since it clarifies what data `feed_urls()` is operating on.